tests/views/test_collection_exercise.py

```diff
@@ -1263,6 +1273,7 @@ def test_update_collection_exercise_userdescription_success(self, mock_request):
 
     @requests_mock.mock()
     def test_update_collection_exercise_details_fail(self, mock_request):
+        sign_in_with_permission(self, mock_request, user_permission_surveys_edit_json)
```

Comment:
Do we sign_in for each, maybe put it in the init?

Reply:
Not all of them have sign in, this is just to test the permissions are working correctly and allow us to test all the functionality, it could be split into two classes